cmd/placement/cmd/placement.go

```diff
@@ -30,12 +33,17 @@ import (
 
 	kcpkubernetesinformers "github.com/kcp-dev/client-go/informers"
 	kcpkubernetesclientset "github.com/kcp-dev/client-go/kubernetes"
+	apisv1alpha1 "github.com/kcp-dev/kcp/pkg/apis/apis/v1alpha1"
+	"github.com/kcp-dev/kcp/pkg/apis/third_party/conditions/util/conditions"
+	kcpclientsetnoncluster "github.com/kcp-dev/kcp/pkg/client/clientset/versioned"
 	kcpclientset "github.com/kcp-dev/kcp/pkg/client/clientset/versioned/cluster"
 	kcpinformers "github.com/kcp-dev/kcp/pkg/client/informers/externalversions"
 	kcpfeatures "github.com/kcp-dev/kcp/pkg/features"
 
 	placementoptions "github.com/kcp-dev/edge-mc/cmd/placement/options"
 	edgeclient "github.com/kcp-dev/edge-mc/pkg/client"
+	edgeclientset "github.com/kcp-dev/edge-mc/pkg/client/clientset/versioned/cluster"
```

Comment:
You might want to put "cluster" in the name of this import, to help keep its meaning explicit.
Or you might not, there is so much verbosity here.

Reply:
Lots of verbosity. So another thing is done to distinguish this, I checked the imports and found that clientset is mostly 'clustered', so I put a suffix 'noncluster' on the single import of the 'nonclustered' clientset.